Dockerfile.build

```diff
@@ -0,0 +1,8 @@
+FROM openshift/origin-release:golang-1.7
+MAINTAINER Federico Simoncelli <fsimonce@redhat.com>
+
+COPY .  /go/src/github.com/openshift/image-inspector
+
+RUN GOBIN=/usr/bin \
+    CGO_ENABLED=0 \
+    go install -a -installsuffix cgo /go/src/github.com/openshift/image-inspector/cmd/image-inspector.go
```

Comment:
It produces a fully statically linked binary that can be dropped inside an empty image (we could even have a 8MB image if it wasn't for the scanner... :blush: )